README.md

```diff
@@ -50,18 +47,26 @@ is just configure and deploy the OpenTelemetry Service separately. The
 OpenTelemetry Service will then automatically collect traces and metrics and
 export to any backend of users' choice.
 
-Currently the OpenTelemetry Service consists of two components, [OpenTelemetry
-Agent](#opentelemetry-agent) and [OpenTelemetry Collector](#opentelemetry-collector).
+Currently the OpenTelemetry Service consists of a single binary and two
+deployment methods:
+
+1. [Agent](#opentelemetry-agent) running with the application or on the same host as the application
+2. [Collector](#opentelemetry-collector) running as a standalone application
```

Comment:
suggestion, somthing along the lines: "typically as a layer of egress with larger buffers and higher resource usage".